components/sup/src/manager/service_updater.rs

```diff
@@ -278,6 +282,31 @@ impl ServiceUpdater {
     }
 }
 
+/// Represents how far apart checks for updates to individual services
+/// are, in milliseconds.
+#[derive(Debug, Clone, Copy, PartialEq, PartialOrd, Eq)]
+struct ServiceUpdatePeriod(u64);
+
+impl Default for ServiceUpdatePeriod {
+    fn default() -> Self {
+        ServiceUpdatePeriod(MIN_ALLOWED_PERIOD)
```

Comment:
```suggestion
        ServiceUpdatePeriod(*MIN_ALLOWED_PERIOD)
```
If you make the `u64` → `Duration` change